exercises/anagram/canonical-data.json

```diff
@@ -11,119 +11,153 @@
     {
       "description": "no matches",
       "property": "anagrams",
-      "subject": "diaper",
-      "candidates": ["hello", "world", "zombies", "pants"],
+      "input": {
+        "subject": "diaper",
+        "candidates": ["hello", "world", "zombies", "pants"
+        ]
```

Comment:
`]` should be at the end of the previous line.

Reply:
Ah, missed that somehow.

Reply:
@rpottsoh Fixed it!